test/module/irohad/logger/logger_test.cpp

```diff
@@ -14,28 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include <logger/logger.hpp>
 #include <gtest/gtest.h>
-#include <iostream>
-
-#include <vector>
-#include <string>
+#include <logger/logger.hpp>
 
-std::vector<std::string> split(std::string str, const std::string& delim) {
-    std::vector<std::string> res;
-    int delim_size = delim.size();
-    while(str.find(delim.c_str()) != std::string::npos){
-        res.push_back(str.substr(0,str.find(delim.c_str())));
-        str = str.substr(str.find(delim.c_str()) + delim_size);
-    }
-    res.push_back(str);
-    return res;
-}
+TEST(LoggerTest, getLoggerTest) {
```

Comment:
Spdlog provide concurrent logging as well. What the purpose to test library in those way?